flepimop/gempyor_pkg/src/gempyor/compartments.py

```diff
@@ -9,11 +27,25 @@
 
 from .utils import config, Timer, as_list
 from .shared_cli import config_files_argument, config_file_options, parse_config_files, cli
+from typing import Union
```

Comment:
This is pre-3.10 syntax, we only support 3.10 and 3.11 so we don't need this. https://docs.python.org/3/library/typing.html#typing.Union